docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -33,20 +45,24 @@ and then reactivating them in small batches at staggered intervals. This
 ensures {kib} does not attempt to run all the rules at the same time.
 * Consider adding another {kib} instance to your environment.
 
+[float]
+[[troubleshoot-ingestion-pipeline-delay]]
+==== Troubleshoot ingestion pipeline delay
+
 Even if your rule runs at its scheduled time, there might still be missing alerts if your ingestion pipeline delay is greater than your rule interval + additional look-back time. Prebuilt rules have a minimum interval + additional look-back time of 6 minutes in {stack} version >=7.11.0. To avoid missed alerts for prebuilt rules, use caution to ensure that ingestion pipeline delays remain below 6 minutes.
 
 In addition, use caution when creating custom rule schedules to ensure that the specified interval + additional look-back time is greater than your deployment's ingestion pipeline delay.
 
-You can eliminate the risk of missed alerts due to ingestion pipeline delay by specifying the `Timestamp override` field value to `event.ingested` in <<rule-ui-advanced-params, advanced settings>> during rule creation or editing. The detection engine uses the value from the `event.ingested` field as the timestamp when executing the rule.
+You can mitigate the risk of missed alerts due to ingestion pipeline delay by specifying the `Timestamp override` field value to `event.ingested` in <<rule-ui-advanced-params, advanced settings>> during rule creation or editing. The detection engine uses the value from the `event.ingested` field as the timestamp when executing the rule.
 
-For example, if an event occurred at 10:01, and due to a 9-minute ingestion pipeline delay, was ingested into {es} at 10:10, a rule created to detect that event, with an interval + additional look-back time of 6 minutes, even if executing at 10:12 -- 11 minutes after the event occurred -- would still detect the event since the `event.ingested` timestamp was only two minutes earlier, well within the 6-minute rule query.
+For example, let's assume an event occurred at 10:00, but it wasn't ingested into {es} until 10:10 due to an ingestion pipeline delay. If we've created a rule to detect that event with an interval + additional look-back time of 6 minutes, and the rule executes at 10:12, it would still detect the event because the `event.ingested` timestamp was from 10:10, only 2 minutes before the rule executed and well within the rule's 6-minute interval + additional look-back time.
```

Comment:
```suggestion
For example, say an event occurred at 10:00 but wasn't ingested into {es} until 10:10 due to an ingestion pipeline delay. If you created a rule to detect that event with an interval + additional look-back time of 6 minutes, and the rule executes at 10:12, it would still detect the event because the `event.ingested` timestamp was from 10:10, only 2 minutes before the rule executed and well within the rule's 6-minute interval + additional look-back time.
```